_data/package_index.yml

```diff
@@ -487,6 +487,13 @@
   tags: back propagation coarray
   version: none
 
+- name: ParaMonte
+  github: cdslaborg/paramonte
+  description: ParaMonte is a well-tested general-purpose open-source high-performance MPI/Coarray-parallel Monte Carlo simulation library implemented in the 2018-standard-complaint Fortran programming language, currently with interfaces to the C/C++/Fortran/MATLAB/Python programming languages.
```

Comment:
```suggestion
  description: A general-purpose high-performance MPI/Coarray-parallel Monte Carlo simulation library implemented in Fortran 2018 with interfaces to C/C++/Fortran/MATLAB/Python 
```

Reply:
Thanks for the corrections, @LKedward. The revised description is indeed far better.